### PR TITLE
Replace Hashid with Expose

### PR DESCRIPTION
**Pull request by dereuromark**
https://github.com/dereuromark/cakephp-expose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,7 +345,8 @@ Additional lists you might find useful:
 - [BryanCrowe/EncryptedType](https://github.com/bcrowe/cakephp-encrypted-type) - Simple solution that enables Encrypted data types to be stored in the database.
 - :strawberry: [Captcha plugin](https://github.com/dereuromark/cakephp-captcha) - Simple, unobstrusive and extendable captcha solution providing by default an image based math captcha.
 - [CipherBehavior plugin](https://github.com/adayth/cakephp-cipher-behavior) - Cipher your entities data with this behavior. Encryption is done at PHP level using CakePHP Security class.
-- [Muffin/Obfuscate plugin](https://github.com/usemuffin/obfuscate) - Primary key obfuscation using HashIds, Optimus, Tiny and/or custom obfuscation strategies.
+- :strawberry: [Expose plugin](https://github.com/dereuromark/cakephp-expose) - Expose entities through additional UUIDs instead of their AIID primary keys to obfuscate those IDs and data associated with these numerically ordered values.
+- [Muffin/Obfuscate plugin](https://github.com/usemuffin/obfuscate) - Primary key obfuscation/shortening using HashIds, Optimus, Tiny and/or custom obfuscation strategies.
```
Comment (jadb) [inline README.md]: Added UUIDs to the list.

```suggestion
- [Muffin/Obfuscate plugin](https://github.com/usemuffin/obfuscate) - Primary key obfuscation/shortening using UUIDs, HashIds, Optimus, Tiny and/or custom obfuscation strategies.
```

Comment (dereuromark) [reply README.md]: At this point this is not true, so I dont think we should add this in here.

Comment (jadb) [reply README.md]: How isn't it? 

You mean because [this](https://github.com/UseMuffin/Obfuscate/pull/16) wasn't merged yet?

Comment (dereuromark) [reply README.md]: Jep, but then again probably only a matter of days.

**Comment by dereuromark** [comment]
Are we able to proceed?

**Pull request merged**